CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Pinned CI reusable actions to specific commit hashes ([#98](https://github.com/arup-group/gelato/issues/98))
```

Comment:
It's not worth making a release just for this change; it's not user-facing. One could argue that it doesn't even belong in the changelog, but I've erred on the side of caution and added it anyway.